Reworded some stuff

KTrain5169 opened the pull request:
Reworded items:
* **Skyclient says** Game has crashed! -> **The game said**
* **Go ahead and** describe your problem so we can help you. -> **Please**
* Click the copy **button** and paste it here. -> **symbol (next to the symbol with an arrow pointing directly up)**
* If you **see** a "View Crash Report" button: -> **saw** & added "in the launcher:"
* If you **don't**: -> **didn't see the button**
* Go to the logs folder and upload the file called **`latest.log`** -> **`latest` or `latest.log`**

KTrain5169 committed:
Clarified the Launcher step

KTrain5169 committed:
Rename Skyclient crash button name & slightly reworded the Something else button response

RuinTD commented on src/interaction-handlers/ticket/categorize.ts:
```diff
@@ -54,7 +54,7 @@ export class ButtonHandler extends InteractionHandler {
 			return interaction.update({
 				content: `${ticketTypeDesc}
-Go ahead and describe your problem so we can help you.
+Please describe your problem so we can help you.
```
This same original phrasing is used in categorizeCrash. Should that be changed as well?

KTrain5169 replied:
yeah that'd work as well

RuinTD commented on src/interaction-handlers/ticket/categorizeCrash.ts:
```diff
@@ -26,15 +26,16 @@ export class ButtonHandler extends InteractionHandler {
-**If you don't:**
-Go to the \`logs\` folder and upload the file called \`latest.log\`.
+**If you didn't see the button:**
+Go to the \`logs\` folder and upload the file called \`latest\` or \`latest.log\`.
```
Windows and Mac default behavior is to *not* show file extensions. Showing file extensions is considered an advanced setting, so maybe we can just say "upload the log called `latest`". Anyone who turned on file extensions (hopefully a semi-competent user!) will know what this means.

Linux users should be expected to figure this out themselves.

```suggestion
Go to the \`logs\` folder and upload the log called \`latest\`
```

KTrain5169 replied:
I say leave it as it is. Simply because, some people might have turned it on at one point and forgotten. Plus, from some tickets I see, `latest` is shown as `latest.log` but the person asking for help seems to be that they wouldn't be able to connect the dots. I do agree that Linux users should know what this means, however.

Sorry if that was confusing, tl;dr leave it bc we can't hope

RuinTD replied:
I would hope that anyone who sees "log called `latest`" would see `latest.log` and realize that is a "log called `latest`". I'm fine with the wording you chose, but I'd personally rather keep it shorter. I'd like to get another opinion on this.

RuinTD replied:
Or maybe instead of "log called `latest`", we could say "log labelled latest"? In my mind, I feel like that would make the phrasing slightly more ambiguous, leading the user to not take the instructions quite so literally.
```suggestion
Go to the \`logs\` folder and upload the log labelled "latest".
```

RuinTD commented on src/interaction-handlers/ticket/categorizeCrash.ts:
```diff
@@ -26,15 +26,16 @@ export class ButtonHandler extends InteractionHandler {
 		switch (crashType) {
 			case 'CP':
-				await interaction.channel.send(`**Click the copy button and paste it here.**\n${plsBePatientTY}`);
+				await interaction.channel.send(`**Click the copy symbol (next to the symbol with an arrow pointing directly up) and paste it here.**\n${plsBePatientTY}`);
```
This text is a little wordy and confusing. Would this maybe make more sense?

```suggestion
				await interaction.channel.send(`**Click the copy icon on the right side and paste the results here.**\n${plsBePatientTY}`);
```

KTrain5169 replied:
yeah that would, I'll commit that

RuinTD committed:
Change second instance of "Go ahead and" to "Please"

RuinTD committed:
Merge branch 'rewrite' into pr/11

KTrain5169 committed:
Update categorizeCrash.ts

As per https://github.com/SkyblockClient/SkyAnswers/pull/11/files/b0ff7488dda3d484329531150fb69d9a2feab70f#r1606075975
